integration_tests/mfa-change-demo/README.md

```diff
@@ -0,0 +1,43 @@
+# Description
+
+Feature store demo.
+
+We use 'simulators' to simulate data input.
+
+Then all messages will be sent to the 'server' and written in Kafka -> RisingWave. RisingWave will process the data based on pre-defined operations.
+
+We also utilize the 'simulator' to simulate user queries to our 'feature'. The 'server' will receive requests -> query data -> and return results.
+
+If we intend to modify our business logic, we simply need to update the materialized view within our RisingWave by using SQL statements.
+
+## Installation
+
+Run it in local.
+
+1. Start kafka and risingwave. Then create source and mv in risingwave. can run this script
+
+```./run_local.sh```
```

Comment:
I saw that a docker compose file is provided but there is no instruction on how to use it. I think we don't need to provide a way to run the demo without docker. Let's update the README for running the demo with docker.